bin/ini2js.js

```diff
@@ -84,10 +84,15 @@ function iniDecode (str) {
     }
     var key = unsafe(match[2])
     var value = match[3] ? unsafe((match[4] || '')) : true
-    switch (value) {
+    // In EHP and production, ansible translates yml files to ini files. It cast true to "True" and false to "False".
+    // This fix keeps consistency between dev and EHP/prod modes while loading settings.
+    // (Modify inlined dependency is weird but I didn't found another solution)
+    var lowerCaseValue = typeof value === 'string' ? value.toLowerCase() : value
```

Comment:
why `value.toLowerCase()` is desirable ? that's a trap !

Reply:
En effet, j'ai voulu faire plus élégant que d'ajouter des `case: 'True'` mais ça ne fait pas la même chose.
Idéalement il faudrait voir comment est introduit ce cast `true` -> `"True"` lors de la génération du `.ini` depuis le `.yml`.
Ce serait mieux que de modifier le `minimist` inlined.

Reply:
Sachant qu'on ne pourra plus passer `True`, `TRUE` etc comme paramètre ... mais peut-êtres c'est peu important. ^^